src/features.jl

```diff
@@ -164,3 +195,63 @@ type(x) = String(x.type)
 bbox(x::GeoJSONObject) = get(object(x), :bbox, nothing)
 
 Base.show(io::IO, ::MIME"text/plain", x::GeoJSONObject) = show(io, x)
+
+# Adapted from JSONTables.jl jsontable method
+# We can simply use their method as we need the key/valu pairs
+# of the properties field, rather than the main object
+function property_schema(features)
+    # Short cut for concrete eltypes of NamedTuple
+    if features isa AbstractArray{<:NamedTuple} && isconcretetype(eltype(x))
+        f1 = first(x)
```

Comment:
```suggestion
    if features isa AbstractArray{<:NamedTuple} && isconcretetype(eltype(features))
        f1 = first(features)
```

Would be good to have a test for this path.

Reply:
Done. Found a bug in `show` as we were not always converting `nothing` to `missing` for geometries.